### PR TITLE
Tp rn rw

### PR DESCRIPTION
**Pull request by kpslp**
did 2023 rn reformat

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -1,895 +1,295 @@
...
+| **Feature** | *Get* and *Delete* application is now available on **Automation API**. | This new feature on the **Automation API** enables tenants to automate the deletion of applications and retrieve detailed information about a specific application within their environment. |
+| **Enhancement** | Enhanced self-sign up page with eligibility checks. | The enhanced self-sign up page with eligibility checks is now available to all users on the TechPass portal. This feature ensures a smoother onboarding process and improved user guidance based on email domain eligibility. |
+| **Enhancement** | Webhook domain whitelist update in staging environment. | The staging environment now accepts webhooks originating from *wogaa* and *ngrok* domains. Webhooks sent from these domains will be processed and handled within the staging environment. |
+| **Fix** | Service account group fix. | We have fixed the issue where adding a service account with an uppercase email address to groups was not functioning correctly. |
+| **Fix** | Restriction on numbers in first and last names. | Previously, users were able to update their first and last names with numbers. It is now disabled. |
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]: `disabled` sounds more like disabling a feature. how about "This is no longer permitted"?

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -1,895 +1,295 @@
...
+| **Fix** | Fix for Previously, when a significant volume of concurrent API calls was initiated, there was a possibility of receiving inaccurate responses. This inconsistency was caused by a racing condition that occurred during high concurrency scenarios.  We have fixed this issue. |
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]: is the `Fix for` redundant?

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -1,895 +1,295 @@
...
+Frontend version: 11.0.0-20230630.1455 | Backend version: 1.69.0-20230630.1619
 
-## Staging release 24 May 2023
-
-Frontend version: 1.0.0-20230520.0010 | Backend version: 1.62.1-230522.1016
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | *Get* and *Delete* application is now available on **Automation API**. | This new feature on the **Automation API** enables tenants to automate the deletion of applications and retrieve detailed information about a specific application within their environment. |
+| **Enhancement** | Enhanced self-sign up page with eligibility checks. | The enhanced self-sign up page with eligibility checks is now available to all users on the TechPass portal. This feature ensures a smoother onboarding process and improved user guidance based on email domain eligibility. |
+| **Enhancement** | Webhook domain whitelist update in staging environment. | The staging environment now accepts webhooks originating from *wogaa* and *ngrok* domains. Webhooks sent from these domains will be processed and handled within the staging environment. |
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]: can I trouble u to list down the newly added domains? we originally have "gov.sg", "ngrok.io" & "amazonaws.com" whitelisted, we are adding these few more:
- ngrok-free.app
- ngrok-free.dev
- ngrok.app
- ngrok.dev
- wogaa.cloud

Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated @yqyeoh

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated @yqyeoh

File: docs/whats-new/staging-release-notes.md (hunk 1 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated @yqyeoh

File: docs/sign-up-and-onboard-to-techpass.md [hunk 4]
```diff
@@ -6,16 +6,16 @@ This article tells you how to sign-up for TechPass via the TechPass Portal and o
 
 Users with a WOG account and who needs a TechPass account.
 
-?>- Public officers using an **SE-GSIB** device need to submit a [service request](https://go.gov.sg/seed-techpass-support) to create their TechPass account.<br>- Alternatively, users can receive onboarding invitations for TechPass and SEED provisioning, which is optional, via the [**TechBiz Portal**](https://portal.techbiz.suite.gov.sg). For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+?>- Public officers using an **SE-GSIB** device need to submit a [service request](https://go.gov.sg/seed-techpass-support) to create their TechPass account.<br>- Alternatively, users can receive onboarding invitations for TechPass and SEED provisioning, which is optional, via the [**TechBiz portal**](https://portal.techbiz.suite.gov.sg) which is only accessible on a **non-SE GSIB** device. For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
```
Comment (yilunyou) [inline docs/sign-up-and-onboard-to-techpass.md]: 2nd point is quite difficult to read. I presume the intention is to show the alternative route to sign up an account via TB.

if a po is at this page trying to self sign up, routing them to techbiz don't make sense.
I will imagine it is helpful for signing up on behalf (techbiz has the mechanism to track that). so i propose the following change.

```suggestion
?>- Public officers using an **SE-GSIB** device need to submit a [service request](https://go.gov.sg/seed-techpass-support) to create their TechPass account.<br>- Public officers can sign up on behalf of others (public officers/vendors) via [**TechBiz portal**](https://portal.techbiz.suite.gov.sg) for account and optional SEED provisioning. TechBiz portal is accessible only on a **non-SE GSIB** device. For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
```

File: docs/sign-up-and-onboard-to-techpass.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Following are the prerequisites to sign up for TechPass and complete the onboard
 
    ![sign-up-submit](assets/images/onboarding/po-non-se/latest-po-sign-up-non-se-gsib-1.png)
 
-4. Click **Submit** to receive the onboarding invitation email(s). If you have also requested for SEED provisioning, you receive two onboarding invitation emails.
+4. Click **Submit** to receive the onboarding invitation email(s). If you have also requested for SEED provisioning, you will receive a total of two onboarding invitation emails.
```
Comment (yilunyou) [inline docs/sign-up-and-onboard-to-techpass.md]:
```suggestion
4. Click **Submit** to receive the onboarding invitation email(s). If you have also requested for SEED provisioning, you will receive a total of two onboarding emails.
```

File: docs/sign-up-and-onboard-to-techpass.md [hunk 6]
```diff
@@ -135,7 +135,9 @@ Following are the prerequisites to sign up for TechPass and complete the onboard
 2. Read the **Terms of Use** and click **Accept**. You have successfully onboarded to TechPass.
 3. If you had requested for SEED provisioning and is successfully provisioned to you, read the **MDM AUP Policy** and click **Accept**.
 
-?>- Upon accepting the terms and conditions, you are successfully onboarded to TechPass.<br>- If you had requested for SEED to be provisioned, you may proceed to onboard your Internet Device to SEED.<br>- Before you onboard your Internet Device to SEED, see[Prerequisites for onboarding your device to SEED](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/prerequisites-for-onboarding). 
+>**Note**: You are only required to accept the terms and conditions when accessing any applications for the first time.
```
Comment (yilunyou) [inline docs/sign-up-and-onboard-to-techpass.md]: accepting of terms and conditions is compulsory to complete TP acct onboarding. 

```suggestion
>**Note**: You are required to accept the terms and conditions when accessing any applications for the first time.
```

File: docs/whats-new/staging-release-notes.md [hunk 7]
```diff
@@ -1,895 +1,295 @@
...
+Frontend version: 11.0.0-20230630.1455 | Backend version: 1.69.0-20230630.1619
 
-## Staging release 24 May 2023
-
-Frontend version: 1.0.0-20230520.0010 | Backend version: 1.62.1-230522.1016
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | *Get* and *Delete* application is now available on **Automation API**. | This new feature on the **Automation API** enables tenants to automate the deletion of applications and retrieve detailed information about a specific application within their environment. |
+| **Enhancement** | Enhanced self-sign up page with eligibility checks. | The enhanced self-sign up page with eligibility checks is now available to all users on the TechPass portal. This feature ensures a smoother onboarding process and improved user guidance based on email domain eligibility. |
+| **Enhancement** | Webhook domain whitelist update in staging environment. | The staging environment now accepts webhooks originating from *wogaa.cloud*, *ngrok-free.app*, *ngrok-free.dev*, *ngrok.app* and *ngrok.dev* domains. Webhooks sent from these domains will be processed and handled within the staging environment. |
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:13]: a reminder that this point is to be removed from prod rn

**Review by yilunyou — CHANGES_REQUESTED** [review]

File: docs/whats-new/staging-release-notes.md (hunk 7 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md:13]: got it

**Review by yilunyou — APPROVED** [review]
approved

**Pull request merged**